cmd/server.go

```diff
@@ -10,50 +10,58 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var onFlag, offFlag bool
+
 var serverCmd = &cobra.Command{
 	Use:   "server",
 	Short: "Print server list",
 	Long:  `Print all server information`,
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
-
 		out := new(bytes.Buffer)
 		servers, err := client.GetServerList(ctx)
 		if err != nil {
 			panic(err)
 		}
 		var serverinfos [][]string
-		if !jsonFlag {
-			heading := []string{"name", "core", "mem", "power", "uuid"}
-			for _, server := range servers {
-				status := "off"
-				if server.Properties.Power {
-					status = "on"
+		if !onFlag && !offFlag {
+			if !jsonFlag {
+				heading := []string{"name", "core", "mem", "power", "uuid"}
+				for _, server := range servers {
```

Comment:
you do not need the for loop, after fixing the above comment.

Reply:
I don't get what you mean by i don't need the for loop to be honest, i do the for-loop so i can fill rows and columns in the table, can you explain it more in detail?

Reply:
you can ignore it. I did not notice this part of `Print server list`